src/lib/device-logs/backends/loki.ts

```diff
@@ -0,0 +1,188 @@
+import * as _ from 'lodash';
+import * as WebSocket from 'ws';
+import { EventEmitter } from 'events';
+import * as skhema from 'skhema';
+import { JSONSchema6 } from 'skhema/node_modules/@types/json-schema';
```

Comment:
This needs to be changed - the fact that this path works at all is pure chance and we should instead import `@types/json-schema` directly

Reply:
This was done intentionally. The version of `json-schema` types used by `skhema` is different from the version of `json-schema` in `dependencies` causing a TS error.

Reply:
This was removed when I remove `skhema` usage